src/gtk/window.cpp

```diff
@@ -6518,3 +6561,31 @@ void wxWindowGTK::DoThaw()
     if (m_wxwindow && m_wxwindow != m_widget)
         GTKThawWidget(m_wxwindow);
 }
+
+// ----------------------------------------------------------------------------
+// wxWinModule
```

Comment:
Let's rename it to `wxXKBModule` please to make it more clear what it is used for.